src/mappings/mappingHandlers.ts

```diff
@@ -147,6 +149,7 @@ export async function handleEvent(event: SubstrateEvent): Promise<void> {
   await Event.create({
     id: `${blockId}/${eventIdx}`,
     blockId,
+    parentBlockId: `${blockId}`,
```

Comment:
Why do we need this when its the same as the blockId?

Reply:
Referring to PR https://github.com/PolymathNetwork/polymesh-sdk/pull/599/files, in order to map the response from the new middleware as the old one, it was required to add parentBlock. (In sdk while fetching events the gql selected fields includes `parentBlock { datetime }`) 

Seems more of a hack.